traitsui/qt4/tree_editor.py

```diff
@@ -203,7 +203,7 @@ def _selection_changed(self, selection):
         try:
             tree = self._tree
             if not isinstance(selection, str) and isinstance(
-                selection, collections.Iterable
+                selection, collections.abc.Iterable
```

Comment:
There is a (terrible) try-except around this block of code... taking a leap of faith.